### DISCUSSION TITLE
Bug : Unable to edit replies of answers on discussions

### DISCUSSION THREAD

**Discussion by EDM115** [community/community#20696] [Mobile] [answered]
If you replied to a discussion, you can edit it on the web app, but not on the mobile app

**Comment by EDM115** [comment]
Android app  
`v1.73.0-beta (10077)`

**Reply by EDM115** [reply]
Example :  
  
![mobile](https://telegra.ph/file/5d8fee4df107b5df9b546.png)  
  
![web](https://telegra.ph/file/32e52a15fe0c0b670379d.png)

**Comment by iskae** [comment, marked as answer]
Hi @EDM115 👋 

Thanks for reaching out ❤️ 

We would love to solve this issue for all, but unfortunately we were not able to reproduce it on our end 😞 Assuming that this issue is always reproducable for you, can you please check if you still have the necessary permissions for the repository?
Also would it be possible if you are logged in with different accounts on mobile and on web?

Thanks in advance for all your help for making GitHub Mobile better and your time! 🙇

**Reply by EDM115** [reply]
Hey  
  
Actually, it seems to be a minimal issue happening only on 1 discussion  
I'm also unable to recreate it anywhere else

**Reply by iskae** [reply]
Hey @EDM115 👋 

Thanks for your response! I am guessing the discussion that this is happening resides in GitHub organisation, right? As a last resort can you please try signing off & on in our mobile app?

**Reply by EDM115** [reply]
Yep, it worked  
Thanks for your hard work ☺️
